docs/knowledge-base/unable-to-find-requested-software.md

```diff
@@ -0,0 +1,47 @@
+# Unable to Find Requested Software
+
+## Environment
+
+- EuroLinux 7
+- EuroLinux 8
+
+## Issue
+
+An error occurs during a search for missing components. A similar message is
+shown by GNOME Software:
+
+> Unfortunately, the [...] you were searching for could not be found. Please
+> see [...] for more information.
+
+### Missing Package
+
+The package you are searching for is not available in EuroLinux base
+repositories.
+
+### Missing Codec
+
+The codec you are searching for is not available in EuroLinux base
+repositories due to licensing reasons or patent encumbrance.
+
+### Missing Hardware Support 
+
+A kernel module or a driver is not available in EuroLinux base repositories
+due to licensing reasons that prohibit redistribution or is not yet in the
+upstream Linux kernel.
+
+### Missing Font
+
+The font you are searching for is not available in EuroLinux base
+repositories.
+
+### Missing MIME Support
+
+An application that supports operations on your file with its MIME type is not
+available in EuroLinux base repositories due to them being supported
+exclusively by a third party application or an application that runs only on a
+platform different than EuroLinux.
+
+## Resolution
+
+If you would like to request for a certain component availability, please
+[contact us](https://en.euro-linux.com/company/contact/).
```

Comment:
https://github.com/EuroLinux/eurolinux-distro-bugs-and-rfc this link is better

Reply:
Changed